pkgs/development/python-modules/poster3/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy3k
+, paste
+, webob
+, pyopenssl
+}:
+
+buildPythonPackage rec {
+  pname = "poster3";
+  version = "0.8.1";
+  format = "wheel";
```

add a comment about wheel format, it's the last option
```suggestion
  format = "wheel"; # only redistributable available
```